upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template

```diff
@@ -77,6 +77,48 @@ data:
 ---
 apiVersion: apps/v1
 kind: Deployment
+metadata:
+  name: coredns-autoscaler
+  namespace: kube-system
+  labels:
+    k8s-addon: coredns.addons.k8s.io
+    k8s-app: coredns-autoscaler
+    kubernetes.io/cluster-service: "true"
+spec:
+  selector:
+    matchLabels:
+      k8s-app: coredns-autoscaler
+  template:
+    metadata:
+      labels:
+        k8s-app: coredns-autoscaler
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      containers:
+      - name: autoscaler
+        image: k8s.gcr.io/cluster-proportional-autoscaler-{{Arch}}:1.4.0
+        resources:
+            requests:
+                cpu: "20m"
+                memory: "10Mi"
+        command:
+          - /cluster-proportional-autoscaler
+          - --namespace=kube-system
+          - --configmap=coredns-autoscaler
```

Comment:
should this configmap exist?

Reply:
No, the proportional autoscaler creates it itself, that's why the `ClusterRole` has the ability to create configmaps within it. 

https://github.com/kubernetes-incubator/cluster-proportional-autoscaler/issues/16 has been closed due to staleness, so don't see this being resolved soon.